feat:added meta tags

Aniket762 opened the pull request:
Hi @romain-fontugne sir, I have added the meta tags which are supposed to increase the reach. I have also added `robots.txt`  which lets Google bots parse it through. If I had the access I would have created a sitemap.xml file in the Public directory. Since I don't have and wanted to try out with basic feats available as of now I just added the meta tags and robots.txt.

Let me know if you want me to do this in some other way round :)

Closes #34

romain-fontugne commented:
Thanks for the meta tags, that is definitely needed!
I'm not sure about the robots.txt. It is not visible in this repository but our webserver already has one: https://ihr.iijlab.net/robots.txt
If you think that it would be better to have it in here then please add "Disallow: /ihr/api/" (the other conditions in the current robots.txt are outdated). We don't want bot to go through the API, this is putting too much strain on our backend.

Aniket762 commented:
> Thanks for the meta tags, that is definitely needed! I'm not sure about the robots.txt. It is not visible in this repository but our webserver already has one: https://ihr.iijlab.net/robots.txt If you think that it would be better to have it in here then please add "Disallow: /ihr/api/" (the other conditions in the current robots.txt are outdated). We don't want bot to go through the API, this is putting too much strain on our backend.

Sir, I have deleted the `robots.txt` file, I didn't knew that robots.txt was already on the server. I believe in having 2 different robots.txt files would create traffic.  So I have deleted the robots.txt file as of now. Incase you want me to do something more please let me know

romain-fontugne commented:
Thanks!
